### PR TITLE
fix: selectors

### PR DESCRIPTION
**Pull request by alexanderleegs**
This PR fixes several e2e test selectors which were looking for the wrong string/colour. Tested on local against staging.

### CODE REVIEW

File: cypress/e2e/editPage.spec.ts [hunk 1]
```diff
@@ -14,6 +14,8 @@ import {
 // Constants
 const PRIMARY_COLOUR = "rgb(255, 0, 0)"
 const SECONDARY_COLOUR = "rgb(0, 255, 0)"
+const SUCCESSFUL_EDIT_PAGE_TOAST =
```
Comment (seaerchin) [inline cypress/e2e/editPage.spec.ts:17]: could we check to see if the old text was being referenced anywhere else? (or if this has been done already, mark thi  as closed)

**Review by seaerchin — APPROVED** [review]

**Review by harishv7 — APPROVED** [review]

File: cypress/e2e/notifications.spec.ts [hunk 2]
```diff
@@ -12,7 +12,7 @@ import {
   visitE2eEmailTestRepo,
 } from "../utils"
 
-const UNREAD_COMMENTS_BG_COLOR = "rgb(235, 248, 255)"
+const UNREAD_COMMENTS_BG_COLOR = "rgb(248, 250, 254)"
```
Comment (harishv7) [inline cypress/e2e/notifications.spec.ts:15]: just to check - was these color changes from design system upgrade?

File: cypress/e2e/editPage.spec.ts (hunk 1 above)
Comment (alexanderleegs) [reply cypress/e2e/editPage.spec.ts]: It was, changed in https://github.com/isomerpages/isomercms-frontend/pull/1399/commits/6445e6b05f8b56b2c228ec8fa48e72be84375c65, and also modified other references to all draw from the same source

File: cypress/e2e/notifications.spec.ts (hunk 2 above)
Comment (alexanderleegs) [reply cypress/e2e/notifications.spec.ts:15]: I think so, @seaerchin can confirm

Comment (seaerchin) [reply cypress/e2e/notifications.spec.ts:15]: Screenshot 2023-08-08 at 1 57 22 PM

`blue.50` token here got changed to isomer specific token

File: cypress/e2e/editPage.spec.ts (hunk 1 above)
Comment (seaerchin) [reply cypress/e2e/editPage.spec.ts]: i don't have anything else alr, feel free to merge

**Pull request merged**